ngraph/frontend/generic/include/frontend_manager/frontend_manager.hpp

```diff
@@ -0,0 +1,364 @@
+// Copyright (C) 2018-2021 Intel Corporation
```

Comment:
not a new file, why isn't the diff displayed?
Are you sure about the target branch for merge?` openvinotoolkit:master`

Reply:
from the 'openvino master' point of view it is actually a new file :-)
This is a generic part (not any-frontend specific) which can be merged to master without affecting anyone